[2025-05-21T22:24:44.892+0000] {logging_mixin.py:154} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T22:24:36.089852+00:00/task_id=transform_api permission to 509
[2025-05-21T22:24:44.893+0000] {logging_mixin.py:154} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T22:24:36.089852+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T22:24:36.089852+00:00/task_id=transform_api'
[2025-05-21T22:24:44.913+0000] {logging_mixin.py:154} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T22:24:36.089852+00:00/task_id=transform_api permission to 509
[2025-05-21T22:24:44.913+0000] {logging_mixin.py:154} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T22:24:36.089852+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T22:24:36.089852+00:00/task_id=transform_api'
[2025-05-21T22:24:44.940+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dimensional_model.transform_api manual__2025-05-21T22:24:36.089852+00:00 [queued]>
[2025-05-21T22:24:44.946+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dimensional_model.transform_api manual__2025-05-21T22:24:36.089852+00:00 [queued]>
[2025-05-21T22:24:44.946+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-05-21T22:24:44.956+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform_api> on 2025-05-21 22:24:36.089852+00:00
[2025-05-21T22:24:44.961+0000] {standard_task_runner.py:57} INFO - Started process 201 to run task
[2025-05-21T22:24:44.963+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dimensional_model', 'transform_api', 'manual__2025-05-21T22:24:36.089852+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/etl_load_postgres_dag.py', '--cfg-path', '/tmp/tmpt4q_gdti']
[2025-05-21T22:24:44.965+0000] {standard_task_runner.py:85} INFO - Job 5: Subtask transform_api
[2025-05-21T22:24:44.999+0000] {logging_mixin.py:154} INFO - Changing /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T22:24:36.089852+00:00/task_id=transform_api permission to 509
[2025-05-21T22:24:44.999+0000] {logging_mixin.py:154} INFO - Failed to change /opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T22:24:36.089852+00:00/task_id=transform_api permission to 509: [Errno 1] Operation not permitted: '/opt/airflow/logs/dag_id=etl_dimensional_model/run_id=manual__2025-05-21T22:24:36.089852+00:00/task_id=transform_api'
[2025-05-21T22:24:45.001+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_dimensional_model.transform_api manual__2025-05-21T22:24:36.089852+00:00 [running]> on host 55c1c6b5908a
[2025-05-21T22:24:45.056+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_dimensional_model' AIRFLOW_CTX_TASK_ID='transform_api' AIRFLOW_CTX_EXECUTION_DATE='2025-05-21T22:24:36.089852+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-21T22:24:36.089852+00:00'
[2025-05-21T22:24:45.076+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-05-21T22:24:45.084+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_dimensional_model, task_id=transform_api, execution_date=20250521T222436, start_date=20250521T222444, end_date=20250521T222445
[2025-05-21T22:24:45.135+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-21T22:24:45.154+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
